### ISSUE TITLE
Add support for GriefPrevention Plus? & Questions.

### ISSUE THREAD

**Issue by Zilacon** [SBPrime/BlocksHub#13]
GriefPreventionPlus is a optimized and lightweight version of GP which is coded by Kai.
https://github.com/KaiKikuchi/GriefPreventionPlus
http://kaikk.net/mc/

Also mind explaining in more detail what this plugin does exactly? From what i thought it was is a modded block api for vanilla block loggers. Not sure if thats correct or not which is why im asking.

**Comment by SBPrime** [comment]
# Please provide a short description of the plugins you want me to add to BH
# What type of plugins are those? Block loggers or building access control?

This plugin is simply an API that provides common functions to check if you can build in a provided location, and forward block changes to block logger plugins. Its basically a multi block logger and a multi block access control. If you add support for it in your plugin you get support for all the supported block loggers and block access controllers. It is similar to vault but for block loggers and block access controls, common API for many types of plugins.

**Comment by Zilacon** [comment]
Its GriefPrevention, just more lightweight and faster. Its a claiming plugin to prevent griefing and raiding and such. You already have a BlockHub-GriefPrevention version but it does not recognize GPP to be GriefPrevention. So just updating BlockHub-GriefPrevention to support both the official GriefPrevention and GriefPreventionPlus would work.

That the only plugin i wanted added.

And is this similar to ForgePermittor in a way? ForgePermittor is a plugin that checks for blockplace, blockbreak, entitydamage, etc events and sends them to the plugins to protect. Basically it makes sure that the vanilla plugins know what modded blocks are and its supposed to protect them.

**Comment by SBPrime** [comment]
Not exactly. As i said before its API for other plugins. The plugin does not handle any block place/break. It forwards block place events from plugins that use BH to all the supported access controller plugins and checks if the block can be placed. In addition it sends the block place/break data to all block logger plugins. The plugin works only for plugins that place blocks and actually use the BH API not for blocks placed by the player.

**Comment by Zilacon** [comment]
So its been 3 months and there is still no compatibility with GP+?
Also i put your plugin side by side with asyncworldedit (in mods folder) to get it to log to prism (yes i have the prism addon for it) and it still doesn't show any logged information for world edits.

In the PRISM config world edit logging is on but its not compatible with asyncworld edit, i figured getting blockshub on here would solve that.
